packages/nimble-components/src/wafer-map/types.ts

```diff
@@ -30,6 +30,7 @@ export interface WaferMapDie {
     value: string;
     x: number;
     y: number;
+    metadata?: unknown;
```

Comment:
We need to document somewhere, maybe a comment in this type, or in the storybook (or both) that this property is not used by the wafer-map and is for the user to optionally store their own arbitrary metadata.

Reply:
applied